### ISSUE TITLE
I can not export a unsigned onnx model using pytorch-quantization

### ISSUE THREAD

**Issue by Meize0729** [NVIDIA/TensorRT#2378]
I have some questions about Pytorch-Quantization provided by tensorRT.

   When I want to use a unsigned=True quantization scheme for the activation of conv2d, I find that the zero-point is fixed at 0 after I export a onnx

model.I think the reason is following below:

def _fb_fake_quant(self, inputs, amax): """Native pytorch fake quantization."""  logging.log_first_n(logging.WARNING, "Use Pytorch's 
                                native experimental fake quantization.", 1)
    bound = (1 << (self._num_bits - 1 + int(self._unsigned))) - 1  # To be consistent with ONNX, full range is used. e.g. range is [-128, 
    127] in int8  if amax.numel() == 1:
        outputs = torch.fake_quantize_per_tensor_affine(
            inputs, amax.item() / bound, 0,  -bound - 1 if not self._unsigned else 0, bound) else:
        amax_sequeeze = amax.squeeze().detach() if len(amax_sequeeze.shape) != 1: raise TypeError("Pytorch's native quantization 
        doesn't support multiple axes")
        quant_dim = list(amax.shape).index(list(amax_sequeeze.shape)[0])
        scale = amax_sequeeze / bound
        outputs = torch.fake_quantize_per_channel_affine(
            inputs, scale.data, torch.zeros_like(scale, dtype=torch.int32).data, quant_dim,  -bound - 1 if not self._unsigned else 0, bound) return outputs

look at zhis place,No matter what the zero point is，it will be 0 after this sentence.Is it a bug or not? How can I do when I want to export a  unsigned onnx model?

**Comment by zerollzeng** [comment]
I think it's a TRT requirement, we only support zero point is 0 now, @ttyio correct me if I'm wrong

**Comment by nvpohanh** [comment]
yeah, TRT doesn't support unsigned quantization

**Comment by ttyio** [comment]
Closing since no activity for more than 3 weeks, please reopen if you still have question, thanks!
